models/ContainerPage.php

```diff
@@ -120,10 +120,11 @@ public function reformatFilter($title)
 	 */
     public function purifyFilter($html)
     {
+	    $settings = new SettingsForm();
 	    $purifierConfig = [
-			    'HTML.Allowed' => '*[style],*[class],div,p,br,b,strong,i,em,u,s,a[href|target],ul,li,ol,span,h1,h2,h3,h4,h5,h6,sub,sup,blockquote,pre,img[src|alt],hr,font[size|color]',
+			    'HTML.Allowed' => $settings->htmlContainerPageAllowedHTML,
```

Comment:
You should fix fix indentation in $purifierConfig [ ]

Reply:
:+1: